simulation.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: page
+title: Second analysis steps
+subtitle: Generating signal decays
+minutes: 60
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Understand how a signal decay sample is produced in the LHCb framework
+> * Produce generator level Monte Carlo, print the decay tree and produce nTuples
+> * Read a DecFile and understand what it produces, including generator level cuts
+> * Filter a simulated sample to reduce disk space needed
+
+## What is Gauss?
+
+The LHCb simulation framework which steers the creation of simulated events and interfaces to multiple external applications. Most commonly, an event is created via the following procedure:
+
+1. The `ProductionTool` (Pythia, GenXicc, ...) generates an event with the required signal particle. Either by generating minimum bias events until a matching particle is found or by ensuring one is produced in every event.
+2. The signal particle is decayed using the `DecayTool` (EvtGen) to the desired final state, all remaining unstable particles are decayed independently.
+3. The signal and its decay products might be required to pass generator level cuts implemented as a `CutTool`.
+4. Particles are transported through the detector simulation.
+
+> ## Things to remember {.callout}
+>
+> 1. The detector simulation is the **__by far__** most time consuming step (minutes). So make sure your generator cuts remove events you cannot possible reconstruct or select later on. Additional options are available to increase the speed, please talk to your MC liaisons!
```

Comment:
* "is the by far" → "is by far the"
* "(minutes)" → "(minutes, compared to seconds for the generation)"
* "possible reconstruct" → "possibly reconstruct"